cores/esp8266/umm_malloc/umm_malloc.cpp

```diff
@@ -646,6 +761,10 @@ unsigned short int umm_numblocks = 0;
 #define UMM_PFREE(b)  (UMM_BLOCK(b).body.free.prev)
 #define UMM_DATA(b)   (UMM_BLOCK(b).body.data)
 
+/*
```

Comment:
This should likely be pursued upstream at the umm repo.

Reply:
I made this comment in case somebody understood better what was going on. It doesn't look like it is being built. Don't know if it is some kind of offline test code. I wondered if maybe it predated the addition of the critical section. I added a note in case someday somebody tries to turn it on.

Reply:
I understand. My point is that we don't own this code, it is a copy taken from the repo of the original author, so making a comment in our copy doesn't help, unless the author were to abandon the code and we were to take ownership of our copy. 
The comment should be made at the author's repo instead, so that it can be addressed.
All of this doesn't mean remove this, just that there is additional action that needs to be taken.